<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyDialogClass</class>
 <widget class="QDialog" name="PropertyDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PropertyDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="diaVL">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gslxGB">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string>钢束类型</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formL">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetFixedSize</enum>
          </property>
          <property name="horizontalSpacing">
           <number>100</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="steelNameLB">
            <property name="text">
             <string>钢束名称：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="steelNameLE"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="steelTypeLB">
            <property name="text">
             <string>钢束类型：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="steelTypeCB"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="materialLB">
            <property name="text">
             <string>材料：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="materialCB"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="steelAreaLB">
            <property name="text">
             <string>钢束总面积：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="gszmjHL">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="steelAreaLE">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="areaUnoinLB">
              <property name="text">
               <string>mm²</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="steelAreaTB">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="cubeDiameterLB">
            <property name="text">
             <string>导管直径：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="dgzjHL">
            <item>
             <widget class="QLineEdit" name="cubeDiameterLE">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="diameterUnionLB">
              <property name="text">
               <string>mm</string>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="steelRelaxCoefLB">
            <property name="text">
             <string>钢筋松弛系数：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="steelRelaxCoefCB"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="steelFpkLB">
            <property name="text">
             <string>预应力钢筋抗拉强度标准值（fpk）：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="fpkHL">
            <item>
             <widget class="QLineEdit" name="steelFpkLE">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="fpkUnionLB">
              <property name="text">
               <string>N/mm²</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="steelfCoefLB">
            <property name="text">
             <string>预应力钢筋与管道壁的摩擦系数：</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="steelfCoefLE">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="steeldCoefLB">
            <property name="text">
             <string>管道每米局部偏差的摩擦影响系数：</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="pcxsHL">
            <item>
             <widget class="QLineEdit" name="steeldCoefLE">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="diffUnionLB">
              <property name="text">
               <string>1/mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="xHL">
          <item>
           <widget class="QGroupBox" name="jshlGB">
            <property name="title">
             <string>锚具变形、钢筋回缩和接缝压缩值</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>100</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="startHL">
               <item>
                <widget class="QLabel" name="startLB">
                 <property name="text">
                  <string>开始：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="startLE"/>
               </item>
               <item>
                <widget class="QLabel" name="startUnionLB">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="finalHL">
               <item>
                <widget class="QLabel" name="finalLB">
                 <property name="text">
                  <string>结束：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="finalLE"/>
               </item>
               <item>
                <widget class="QLabel" name="finalUnionLB">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="zjlxGB">
            <property name="title">
             <string>粘结类型</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="glueRB">
               <property name="text">
                <string>粘结</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="unglueRB">
               <property name="text">
                <string>无粘结</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="btnHL">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>300</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="confirmBtn">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
